### ISSUE TITLE
`createKeaStore` was not found in '../scene/store'

### ISSUE THREAD

**Issue by adam187** [keajs/kea#17]
Hi,

I tried to install `kea` and configure it like described in step 1 https://kea.js.org/guide/installation but i'm getting error:

```
./node_modules/kea/src/logic/_deprecated.js
91:9-12 "export 'createKeaStore' (imported as 'cks') was not found in '../scene/store'
```

This happened using version `0.19.12`.
I'm using `module` main field https://github.com/keajs/kea/blob/master/package.json#L8 via `webpack` + `babel`

Error is coming from https://github.com/keajs/kea/blob/master/src/logic/_deprecated.js#L3
and export is actually missing in https://github.com/keajs/kea/blob/master/src/scene/store.js
But https://github.com/keajs/kea/blob/master/src/scene/index.js#L6 is also trying to export 

So it looks like `createKeaStore` is missing in ./src/scene/store.js`

**Comment by mariusandra** [comment]
Hey, please try `0.19.13` :)

**Comment by mariusandra** [comment]
Hi, for a somewhat longer explanation, I just released `0.19.12`, which added the `module` main field to `package.json`, when writing the guide on how to get kea working with `create-react-app`.

I then realised that my imports were too loose and `src/index.js` (`import * from 'kea'`) bundled too many unused functions. So I released `0.19.13` [fixing this issue](https://github.com/keajs/kea/commit/5ff4e81ffc0b056fc041593792b19af26b844bf2#diff-1fdf421c05c1140f6d71444ea2b27638).

So now if you import from `'kea'`, you should not see the errors you saw... at least in theory :)

However, it's true that `createKeaStore` is missing altogether. So I released `0.19.14` which removes any mention of it.

This `createKeaStore` was an old and smelly way of injecting `kea` to the store. I completely removed it in the beginning of the 0.19 series.

Thanks for finding and reporting the issue! 👍

**Comment by adam187** [comment]
@mariusandra I can confirm that `0.19.14` fixed issue :)
